packages/stream_video_flutter/lib/src/call_screen/lobby_video.dart

```diff
@@ -112,83 +112,85 @@ class _StreamLobbyVideoState extends State<StreamLobbyVideo> {
-                        alignment: Alignment.bottomLeft,
-                        child: Column(
-                          crossAxisAlignment: CrossAxisAlignment.start,
-                          mainAxisSize: MainAxisSize.min,
-                          children: [
-                            StreamParticipantLabel(
-                              isAudioEnabled: microphoneEnabled,
-                              isSpeaking: false,
-                              participantName: currentUser.name,
-                            ),
-                          ],
+                      );
+                    }
+
+                    return Stack(
+                      children: [
+                        if (cameraEnabled)
+                          VideoTrackRenderer(
+                            mirror: true,
+                            videoTrack: _cameraTrack!,
+                            placeholderBuilder: placeHolderBuilder,
+                          )
+                        else
+                          placeHolderBuilder(context),
+                        Align(
+                          alignment: Alignment.bottomLeft,
+                          child: Column(
+                            crossAxisAlignment: CrossAxisAlignment.start,
+                            mainAxisSize: MainAxisSize.min,
+                            children: [
+                              StreamParticipantLabel(
+                                isAudioEnabled: microphoneEnabled,
+                                isSpeaking: false,
+                                participantName: currentUser.name,
+                              ),
+                            ],
+                          ),
                         ),
-                      ),
-                    ],
-                  );
-                },
+                      ],
+                    );
+                  },
+                ),
               ),
             ),
           ),
-        ),
-        const SizedBox(height: 16),
-        Row(
-          children: [
-            CallControlOption(
-              icon: microphoneEnabled
-                  ? const Icon(Icons.mic_rounded)
-                  : const Icon(Icons.mic_off_rounded),
-              iconColor: microphoneEnabled ? null : theme.optionOffIconColor,
-              backgroundColor:
-                  microphoneEnabled ? null : theme.optionOffBackgroundColor,
-              onPressed: toggleMicrophone,
-            ),
-            CallControlOption(
-              icon: cameraEnabled
-                  ? const Icon(Icons.videocam_rounded)
-                  : const Icon(Icons.videocam_off_rounded),
-              iconColor: cameraEnabled ? null : theme.optionOffIconColor,
-              backgroundColor:
-                  cameraEnabled ? null : theme.optionOffBackgroundColor,
-              onPressed: toggleCamera,
-            ),
-          ],
-        ),
-      ],
+          const SizedBox(height: 16),
+          Row(
```

Comment:
maybe we should also center the childrens. Also make sure to use `mainAxisSize: MainAxisSize.min`

Reply:
After some discussions offline we've decided that centering is indeed the preferred way and the current layout could be considered a bug.